slack-invite/models/team.rb

```diff
@@ -62,6 +64,11 @@ def inform!(message)
     end
   end
 
+  def workspace_url
+    team_info = slack_client.team_info
+    team_info.to_hash['team']['url']
```

Comment:
I would break this into two methods:

```
def team_info
    @team_info ||= slack_client.team_info
end

def workspace_url
   team_info&.team.url
end
```

(See if `to_hash...` is unnecessary and whether the above works).